### PR TITLE
chore(deps): Enable renovate automerge for minor and patch bumps

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -33,6 +33,13 @@
 		"npm"
 	],
 	"packageRules": [
+		{
+			"matchUpdateTypes": ["minor", "patch"],
+			"matchCurrentVersion": "!/^0/",
```
Comment (st3iny) [inline renovate.json:38]: Why exclude certain versions?

Comment (ChristophWurst) [reply renovate.json:38]: 0.x is typically not considered production ready, nor is it clearly defined how semver works before 1.x

let's review those manually

same with major bumps as they are likely coming with breaking changes

Comment (ChristophWurst) [reply renovate.json:38]: I can also drop that. I copied the config from the sentry app. And the sentry app copied from text IIRC :see_no_evil:

Comment (st3iny) [reply renovate.json:38]: Fine by me. Many libraries use 0.*.* to mark not-production-ready versions.

**Pull request merged**